web-client/src/constants.tsx

```diff
@@ -63,6 +63,9 @@ export const VERBS = [
   'delete',
 ]
 
-export const templateNamespacedResourceRolePrefix = 'template-namespaced-resources___'
+export const resourceSeparator = '___'
```

Comment:
can you use it also [here](https://github.com/sighupio/permission-manager/blob/master/web-client/src/services/rolebindingCreateRequests.tsx#L147) and [here](https://github.com/sighupio/permission-manager/blob/master/web-client/src/services/rolebindingCreateRequests.tsx#L168)?